rds.tf

```diff
@@ -9,7 +17,7 @@ resource "aws_db_instance" "rds-db" {
   multi_az                = false
   name                    = "gftclientdb"
   #parameter_group_name     = "mydbparamgroup1" # if you have tuned it
-  password               = var.db_password
+  password               = data.aws_secretsmanager_secret_version.secretVers.secret_string
```

Comment:
should be changed to jsondecode(data.aws_secretsmanager_secret_version.secretVars.secret_string)["password"]